birdhouse/components/monitoring/default.env

```diff
@@ -0,0 +1,28 @@
+# All env in this default.env can be overridden by env.local.
+# All env in this default.env must not depend on any env in env.local.
+
+export GRAFANA_ADMIN_PASSWORD="changeme!"
+export ALERTMANAGER_ADMIN_EMAIL_RECEIVER=""  # "user1@example.com,user2@example.com"
+export SMTP_SERVER=""  # "smtp.example.com:25"
+export ALERTMANAGER_EXTRA_GLOBAL=""
+export ALERTMANAGER_EXTRA_ROUTES=""
+export ALERTMANAGER_EXTRA_INHIBITION=""
+export ALERTMANAGER_EXTRA_RECEIVERS=""
+
+
+
+# add vars only needed to be substituted in templates
+
+VARS="
+  $VARS
+  \$ALERTMANAGER_ADMIN_EMAIL_RECEIVER
+  \$SMTP_SERVER
```

Comment:
+1 on this one
I was wondering why these variable needed to be defined even if monitoring wasn't enabled.
This offers more flexibility, great.